packages/xpetra/src/CrsGraph/Xpetra_EpetraCrsGraph.hpp

```diff
@@ -726,7 +739,19 @@ class EpetraCrsGraphT<int, EpetraNode>
 
 #ifdef HAVE_XPETRA_KOKKOS_REFACTOR
 #ifdef HAVE_XPETRA_TPETRA
-  local_graph_type getLocalGraph () const {
```

Comment:
Same as below (I'm going bottom to top for some reason...) why not keep in `TPETRA_ENABLE_DEPRECATED_CODE` scope?